meta-genivi-dev/meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,34 +1,43 @@
-SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "95aadfb33d95e14030585c3fa2e1afb0e7b743c8"
+DESCRIPTION = "Reference HMI application for GDP"
+HOMEPAGE = "https://github.com/GENIVI/hmi-layout-gdp"
 LICENSE  = "MPL-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=9741c346eef56131163e13b9db1241b3"
 
-DEPENDS = "dbus-c++ systemd wayland-ivi-extension qtquick1 qtdeclarative qtbase ivi-logging"
+SRC_URI = "\
+    git://github.com/GENIVI/hmi-layout-gdp.git \
+    file://gdp-new-hmi.service \
+"
 
-RDEPENDS_${PN} += "qtbase qtsvg"
+SRCREV = "95aadfb33d95e14030585c3fa2e1afb0e7b743c8"
 
 S = "${WORKDIR}/git"
 
-inherit autotools pkgconfig qmake5 systemd
-
-SYSTEMD_AUTO_ENABLE = "enable"
+DEPENDS = "\
+    dbus-c++ \
+    ivi-logging \
+    qtbase \
+    qtdeclarative \
+    qtquick1 \
+    systemd \
+    wayland-ivi-extension \
+"
 
-SRC_URI_append ="\
-    file://gdp-new-hmi.service \
-    "
+RDEPENDS_${PN} = "qtbase qtsvg"
 
-FILES_${PN} += "\
-    ${libdir}/* \
-    /opt/gdp-hmi/bin/gdp-hmi \
-    /usr/share/applications/* \
-    ${systemd_unitdir}/* \
-    /home/* \
-"
+inherit autotools pkgconfig qmake5
 
 do_install_append() {
-        install -d ${D}${libdir}/systemd/user
-        install -m 0444 ${WORKDIR}/gdp-new-hmi.service \
-                        ${D}${libdir}/systemd/user
-        mkdir -p ${D}/home/root/.config/systemd/user/default.target.wants/gdp-new-hmi.service
-	ln -sf /usr/lib/systemd/user/gdp-new-hmi.service ${D}/home/root/.config/systemd/user/default.target.wants/gdp-new-hmi.service
+    install -d ${D}${systemd_user_unitdir}
+    install -p -D ${WORKDIR}/gdp-new-hmi.service ${D}${systemd_user_unitdir}/gdp-new-hmi.service
+    install -d ${D}${sysconfdir}/systemd/user/default.target.wants
+    ln -sf ${systemd_user_unitdir}/gdp-new-hmi.service ${D}${sysconfdir}/systemd/user/default.target.wants
```

Comment:
You're installing the symlink to a different location (/etc).  I know because I just made the same change :)  and I agree... :)
I think this fixes the dependencies again so the startup is correct.

Reply:
I believe that I am just mimicking what `systemd --user enable <service>` would do if executed on target.

But I will check again.

I saw your change and left a comment on it :)